### PR TITLE
Use version 2.0.0 of argus-toolbelt docker image

### PR DESCRIPTION
**Pull request by konraduh**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Hotfix. Version 3 of docker image argus-toolbelt not compatible with this integration. 

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ x ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Development container for Content. Poetry hash 1.11.0
+// Development container for Content. Poetry hash 1b1ee2d03208b85c59c1aaaf0726aa7db0fdf6bb5c0766919268dfb44d47b9b6
```
Comment (merit-maita) [inline .devcontainer/devcontainer.json:1]:
```suggestion
// Development container for Content. Poetry hash 1.11.0
```
it was changed to sdk version intentionally, so please revert it

**Review by merit-maita — CHANGES_REQUESTED** [review]
thanks for your contribution!
please make sure the checks pass

**Comment by merit-maita** [comment]
Hello @konraduh 
can you please address my comment you resolved regarding the hash added, and check the circleCi SetUp Environment that's failing.
thanks!

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (merit-maita) [inline .devcontainer/devcontainer.json:1]:
```suggestion
// Development container for Content. Poetry hash 1.11.0
```

**Comment by merit-maita** [comment]
@konraduh  any update?

**Comment by konraduh** [comment]
Hi,

So sorry, I'm away until next month, I'll see if someone else can deal with
it. I tried adding your suggested edit to the comment.

Sorry to take up your time.

BR,
Konrad

**Comment by merit-maita** [comment]
@konraduh please update me if someone can take it soon, if not, you can close it for now, and reopen it once its ready

**Comment by merit-maita** [comment]
@konraduh i'm closing the pr now, since it seems like its being on-hold, please reopen in case there's some update

**Pull request closed without merging**